plugins/woocommerce/includes/class-wc-privacy-exporters.php

```diff
@@ -416,7 +416,7 @@ public static function customer_tokens_exporter( $email_address, $page ) {
 			)
 		);
 
-		if ( 0 < count( $tokens ) ) {
+		if ( is_array( $tokens ) && 0 < count( $tokens ) ) {
```

Comment:
$tokens is also guaranteed to be an array.

Reply:
done https://github.com/woocommerce/woocommerce/pull/41482/commits/4ea60f952a4ee8165e9c74259411eb6b0f5a74ed